### PR TITLE
User can view all grades - Jay Hoang Nguyen

### PR DESCRIPTION
**Pull request by hoangnguyen9384**
Screen Shot 2020-08-20 at 12 16 35 PM

### CODE REVIEW

File: components/app.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+  handleGetGradesSuccess(grades){
+    this.gradeTable.updateGrades(grades);
+  }
+
+  getGrades(){
+    $.ajax({
+      type:"GET",
+      url:"https://sgt.lfzprototypes.com/api/grades",
+      data:"none",
```
Comment (TimelikeClosure) [inline components/app.js:18]: `data; "none"` sends a request body containing `none`. If you don't wish to send a request body, don't supply a `.data` property.

File: components/grade-table.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class GradeTable {
+  constructor(tableElement){
+    this.tableElement = tableElement;
+
+  }
+  updateGrades(grades){
+    console.log('grades: ', grades);
```
Comment (TimelikeClosure) [inline components/grade-table.js:7]: Please remove `console.log`s when you're done with them.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <tr>
+        <th scope="col">Student Name</th>
+        <th scope="col">Course</th>
+        <th scope="col">Grade</th>
+      </tr>
+    </thead>
+    <tbody>
+
+    </tbody>
+  </table>
```
Comment (TimelikeClosure) [inline index.html:28]: The amount of spaces used to indent children is inconsistent. Please fix this.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <title>Student Grades Table</title>
+</head>
+<body>
+  <header class="student-grade-table m-5">
+      <h2>Student Grade Table</h2>
+  </header>
```
Comment (TimelikeClosure) [inline index.html:13]: To clarify my previous indentation comment, this `<h2>` is indented 4 spaces from the `<header>`'s indent, while the rest of the document uses 2 spaces for each level of indenting.

Comment (hoangnguyen9384) [reply index.html:13]: ok i'll fix it.thanks Tim!

**Pull request merged**